packages/vue/src/components/list/SingleList.jsx

```diff
@@ -1,14 +1,14 @@
 import { Actions, helper } from '@appbaseio/reactivecore';
+import { isEqual } from '@appbaseio/reactivecore/lib/utils/helper';
```

Comment:
Same here.

Reply:
I didn't import it, re-order it to solve `eslint` warning of `absolute imports should be before relative imports`